### PR TITLE
Logging fix in toolshed install manager.

### PR DESCRIPTION
**Pull request by dannon**
Ran into this bug trying to track down an exception I was having installing stuff -- I assume this is the original intent but was just incorrectly implemented.

### CODE REVIEW

File: lib/tool_shed/galaxy_install/install_manager.py [hunk 1]
```diff
@@ -1023,9 +1023,10 @@ def fetch_tool_versions( app, tool_shed_repository ):
         if text:
             return json.loads( text )
         else:
+            log.error("No content returned from tool shed repository version request to %s", url)
             failed_to_fetch = True
     except Exception:
         failed_to_fetch = True
-    if failed_to_fetch:
         log.exception("Failed to fetch tool shed repository verion information.")
```
Comment (martenson) [inline lib/tool_shed/galaxy_install/install_manager.py]: s/verion/version

not sure if worth for release though

Comment (dannon) [inline lib/tool_shed/galaxy_install/install_manager.py]: Yeah, I'm updating PR, saw that myself after I started working on something else :)

File: lib/tool_shed/galaxy_install/install_manager.py [hunk 2]
```diff
@@ -1023,9 +1023,10 @@ def fetch_tool_versions( app, tool_shed_repository ):
         if text:
             return json.loads( text )
         else:
+            log.error("No content returned from tool shed repository version request to %s", url)
             failed_to_fetch = True
     except Exception:
         failed_to_fetch = True
+        log.exception("Failed to fetch tool shed repository version information.")
```
Comment (nturaga) [inline lib/tool_shed/galaxy_install/install_manager.py:1030]: All good. Seems to be moved to the right place.

**Comment by blankenberg** [comment]
:+1:

**Pull request merged**